dev/com.ibm.ws.cdi.1.2_fat/.classpath

```diff
@@ -0,0 +1,174 @@
+	<classpathentry kind="src" path="test-applications/warLibAccessBeansInWar.war/src"/>
+	<classpathentry kind="src" path="test-applications/warLibAccessBeansInWar2.jar/src"/>
+	<classpathentry kind="src" path="test-applications/warLibAccessBeansInWarJar.jar/src"/>
+	<classpathentry kind="src" path="test-applications/webBeansBeansXmlDecorators.war/src"/>
+	<classpathentry kind="src" path="test-applications/webBeansBeansXmlInterceptors.war/src"/>
+	<classpathentry kind="src" path="test-applications/WebListener.war/src"/>
+	<classpathentry kind="src" path="test-applications/withAnnotationsApp.war/src"/>
+	<classpathentry kind="src" path="test-bundles/cdi.helloworld.extension/src"/>
+	<classpathentry kind="src" path="test-applications/deltaspikeTest.war/src"/>
+	<classpathentry kind="src" path="/com.ibm.websphere.javaee.annotation.1.2"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="/ant_build/lib/junit.jar"/>
+	<classpathentry kind="lib" path="/build.sharedResources/lib/hamcrest-all.jar"/>
+	<classpathentry kind="src" path="/fattest.simplicity"/>
+	<classpathentry kind="src" path="/com.ibm.ws.logging"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.ws.org.apache.commons.logging"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.interceptor.1.2"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.jsf.2.2"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.ejb.3.2"/>
+	<classpathentry kind="src" path="/com.ibm.ws.cdi.1.2.weld.impl"/>
+	<classpathentry kind="src" path="/com.ibm.ws.cdi.1.2.interfaces"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.org.osgi.core"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.org.osgi.service.component"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.wsspi.org.osgi.service.component.annotations"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.jms.2.0"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.security"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.cdi.1.2"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.servlet.3.1"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.connector.1.7"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.validation.1.1"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.el.3.0"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.jsp.2.3"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.websphere.javaee.websocket.1.1"/>
```

Comment:
I think that only src entries should be listed here in the .classpath. The rest of the binary dependencies should be dealt with by the bnd plugin.